smrf/tests/test_topo.py

```diff
@@ -12,11 +12,7 @@ class TestLoadTopo(unittest.TestCase):
     @classmethod
     def setUp(self):
         base = os.path.dirname(smrf.__file__)
-        self.test_dir = os.path.abspath(os.path.join(base, '../', 'tests'))
-        tempDir = os.path.join(self.test_dir, 'RME/output')
-
-        if not os.path.exists(tempDir):
-            os.mkdir(tempDir)
+        self.test_dir = os.path.abspath(os.path.join(base, 'tests'))
```

Comment:
Looks like this could be a class method as well and make the folder name part an argument.

Reply:
That would be a good use of `@property`